src/app/public/modules/split-view/split-view.component.spec.ts

```diff
@@ -755,12 +756,28 @@ describe('Split view component', () => {
     it('should pass accessibility', async(() => {
       fixture.componentInstance.ariaLabelForDrawer = 'My drawer';
       fixture.detectChanges();
-      expect(fixture.nativeElement).toBeAccessible();
+      fixture.whenStable();
```

Comment:
`whenStable` returns a `Promise`, so you'll need to wait for it to finish before executing the next line. Also, the code can be a little cleaner if you use the vanilla Javascript `async` instead of the angular `async` test. Our a11y tests usually look something like this...
```

  it('should pass accessibility', async () => {
    fixture.detectChanges();
    await fixture.whenStable();
    await expectAsync(fixture.nativeElement).toBeAccessible();
  });
```
This would apply to your other test as well.